### ISSUE TITLE
Crash on 1.20.6

### ISSUE THREAD

**Issue by UsainSrht** [Khajiitos/ServerCountryFlags#12]
Opened up the server list and randomly it crashed.

crash log:
```
---- Minecraft Crash Report ----
// Daisy, daisy...

Time: 2024-05-01 12:31:57
Description: Rendering screen

java.lang.NoSuchMethodError: 'void net.minecraft.class_500.method_2528(java.util.List)'
	at net.minecraft.class_4267$class_4270.handler$cin000$servercountryflags$render(class_4267.java:662)
	at net.minecraft.class_4267$class_4270.method_25343(class_4267.java:350)
	at net.minecraft.class_350.method_44397(class_350.java:438)
	at net.minecraft.class_350.method_25311(class_350.java:423)
	at net.minecraft.class_350.method_48579(class_350.java:187)
	at net.minecraft.class_339.method_25394(class_339.java:66)
	at net.minecraft.class_437.method_25394(class_437.java:131)
	at net.minecraft.class_500.method_25394(class_500.java:250)
	at net.minecraft.class_437.method_47413(class_437.java:120)
	at net.minecraft.class_757.mixinextras$bridge$method_47413$244(class_757.java)
	at net.minecraft.class_757.wrapOperation$bee000$fabric-screen-api-v1$onRenderScreen(class_757.java:3101)
	at net.minecraft.class_757.method_3192(class_757.java:922)
	at net.minecraft.class_310.method_1523(class_310.java:1320)
	at net.minecraft.class_310.method_1514(class_310.java:888)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_4267$class_4270.handler$cin000$servercountryflags$render(class_4267.java:662)
	at net.minecraft.class_4267$class_4270.method_25343(class_4267.java:350)
	at net.minecraft.class_350.method_44397(class_350.java:438)
	at net.minecraft.class_350.method_25311(class_350.java:423)
	at net.minecraft.class_350.method_48579(class_350.java:187)
	at net.minecraft.class_339.method_25394(class_339.java:66)
	at net.minecraft.class_437.method_25394(class_437.java:131)
	at net.minecraft.class_500.method_25394(class_500.java:250)
	at net.minecraft.class_437.method_47413(class_437.java:120)
	at net.minecraft.class_757.mixinextras$bridge$method_47413$244(class_757.java)
	at net.minecraft.class_757.wrapOperation$bee000$fabric-screen-api-v1$onRenderScreen(class_757.java:3101)

-- Screen render details --
Details:
	Screen name: net.minecraft.class_500
	Mouse location: Scaled: (367, 260). Absolute: (735.000000, 521.000000)
	Screen size: Scaled: (960, 505). Absolute: (1920, 1009). Scale factor of 2.000000
Stacktrace:
	at net.minecraft.class_757.method_3192(class_757.java:922)
	at net.minecraft.class_310.method_1523(class_310.java:1320)
	at net.minecraft.class_310.method_1514(class_310.java:888)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Uptime --
Details:
	JVM uptime: 77.883s
	Wall uptime: 67.817s
	High-res time: 64.243s
	Client ticks: 1247 ticks / 62.350s
Stacktrace:
	at net.minecraft.class_310.method_1587(class_310.java:2548)
	at net.minecraft.class_310.method_54580(class_310.java:983)
	at net.minecraft.class_310.method_1514(class_310.java:908)
	at net.minecraft.client.main.Main.main(Main.java:265)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: vanilla, fabric, appleskin, betterconfig, boatiview, cleardespawn, clientcommands, cloth-config, dynamic_fps, elytra_physics, entity_model_features, entity_texture_features, explosiveenhancement, extremesoundmuffler, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, forcecloseworldloadingscreen, forgeconfigapiport, freecam, immediatelyfast, invmove, iris, lambdynlights, lazydfu, lithium, modmenu, nbttooltip, norefreshscroll, pride, rrls, servercountryflags, serverpingerfixer, sodium, spruceui, tia, truesight, viafabric, viaversion, yet_another_config_lib_v3, zoomify

-- System Details --
Details:
	Minecraft Version: 1.20.6
	Minecraft Version ID: 1.20.6
	Operating System: Windows 11 (amd64) version 10.0
	Java Version: 21.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 361329816 bytes (344 MiB) / 1308622848 bytes (1248 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 8
	Processor Vendor: GenuineIntel
	Processor Name: 12th Gen Intel(R) Core(TM) i3-12100F
	Identifier: Intel64 Family 6 Model 151 Stepping 5
	Microarchitecture: Alder Lake
	Frequency (GHz): 3.30
	Number of physical packages: 1
	Number of physical CPUs: 4
	Number of logical CPUs: 8
	Graphics card #0 name: AMD Radeon RX 6600
	Graphics card #0 vendor: Advanced Micro Devices, Inc.
	Graphics card #0 VRAM (MB): 8176.00
	Graphics card #0 deviceId: VideoController1
	Graphics card #0 versionInfo: 31.0.12027.9001
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.20
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.20
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 28485.42
	Virtual memory used (MB): 15533.61
	Swap memory total (MB): 12288.00
	Swap memory used (MB): 114.75
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		appleskin: AppleSkin 3.0.0+mc1.20.5
		boatiview: Boat Item View Fabric 0.0.5
		cleardespawn: Clear Despawn 1.1.15
		clientcommands: Client Commands 2.8.17
			betterconfig: BetterConfig 1.2.1
			clientarguments: clientarguments 1.8.2
			net_fabricmc_mapping-io: mapping-io 0.5.1
		cloth-config: Cloth Config v14 14.0.126
			cloth-basic-math: cloth-basic-math 0.6.1
		dynamic_fps: Dynamic FPS 3.4.3
		elytra_physics: Elytra Physics 2.0
		entity_model_features: Entity Model Features 2.0.2
		entity_texture_features: Entity Texture Features 6.0.1
			org_apache_httpcomponents_httpmime: httpmime 4.5.10
		explosiveenhancement: Explosive Enhancement 1.2.2-1.20.x
		extremesoundmuffler: Extreme sound muffler 3.45.1
			fiber: fiber 0.23.0-2
		fabric-api: Fabric API 0.97.8+1.20.6
			fabric-api-base: Fabric API Base 0.4.40+80f8cf51ff
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.59+e9d2a72bff
			fabric-biome-api-v1: Fabric Biome API (v1) 13.0.25+be5d88beff
			fabric-block-api-v1: Fabric Block API (v1) 1.0.20+6dfe4c9bff
			fabric-block-view-api-v2: Fabric BlockView API (v2) 1.0.8+80f8cf51ff
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.50+80f8cf51ff
			fabric-client-tags-api-v1: Fabric Client Tags 1.1.12+7f945d5bff
			fabric-command-api-v1: Fabric Command API (v1) 1.2.45+f71b366fff
			fabric-command-api-v2: Fabric Command API (v2) 2.2.24+80f8cf51ff
			fabric-commands-v0: Fabric Commands (v0) 0.2.62+df3654b3ff
			fabric-content-registries-v0: Fabric Content Registries (v0) 8.0.4+b82b2392ff
			fabric-convention-tags-v1: Fabric Convention Tags 2.0.3+7f945d5bff
			fabric-convention-tags-v2: Fabric Convention Tags (v2) 2.0.0+2b43c5c8ff
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.27+80f8cf51ff
			fabric-data-attachment-api-v1: Fabric Data Attachment API (v1) 1.1.15+2a2c66b6ff
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 19.0.6+7f945d5bff
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.68+94793913ff
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.6.8+e9d2a72bff
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.7.6+c5fc38b3ff
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.50+80f8cf51ff
			fabric-item-api-v1: Fabric Item API (v1) 8.1.1+17e985d6ff
			fabric-item-group-api-v1: Fabric Item Group API (v1) 4.0.38+aae0949aff
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.45+80f8cf51ff
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.43+df3654b3ff
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.3.4+c5fc38b3ff
			fabric-loot-api-v2: Fabric Loot API (v2) 3.0.4+97f703daff
			fabric-message-api-v1: Fabric Message API (v1) 6.0.10+109a837cff
			fabric-model-loading-api-v1: Fabric Model Loading API (v1) 1.0.12+80f8cf51ff
			fabric-models-v0: Fabric Models (v0) 0.4.11+9386d8a7ff
			fabric-networking-api-v1: Fabric Networking API (v1) 4.0.8+0dca0349ff
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 15.1.3+c5fc38b3ff
			fabric-particles-v1: Fabric Particles (v1) 4.0.0+c5fc38b3ff
			fabric-recipe-api-v1: Fabric Recipe API (v1) 5.0.3+c5fc38b3ff
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 5.0.15+f1240ba7ff
			fabric-renderer-api-v1: Fabric Renderer API (v1) 3.2.12+97f703daff
			fabric-renderer-indigo: Fabric Renderer - Indigo 1.5.12+80f8cf51ff
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.61+df3654b3ff
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.46+73761d2eff
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.1.3+2c869dedff
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.64+df3654b3ff
			fabric-rendering-v1: Fabric Rendering (v1) 4.2.4+b21c00cbff
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 4.0.1+74e2f560ff
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 1.0.5+c5f2432cff
			fabric-screen-api-v1: Fabric Screen API (v1) 2.0.21+7b70ea8aff
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.3.72+b21c00cbff
			fabric-sound-api-v1: Fabric Sound API (v1) 1.0.21+c5fc38b3ff
			fabric-transfer-api-v1: Fabric Transfer API (v1) 5.1.6+c5fc38b3ff
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 6.0.10+74e2f560ff
		fabric-language-kotlin: Fabric Language Kotlin 1.10.19+kotlin.1.9.23
			org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.9.23
			org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.9.23
			org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.9.23
			org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.9.23
			org_jetbrains_kotlinx_atomicfu-jvm: atomicfu-jvm 0.23.2
			org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.8.0
			org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.8.0
			org_jetbrains_kotlinx_kotlinx-datetime-jvm: kotlinx-datetime-jvm 0.5.0
			org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm: kotlinx-serialization-cbor-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.6.3
			org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.6.3
		fabricloader: Fabric Loader 0.15.10
			mixinextras: MixinExtras 0.3.5
		forcecloseworldloadingscreen: kennytv's epic force close loading screen mod for Fabric 2.2.2
		forgeconfigapiport: Forge Config API Port 20.6.0
			com_electronwill_night-config_core: core 3.6.7
			com_electronwill_night-config_toml: toml 3.6.7
		freecam: Freecam 1.2.4+mc1.20.5
		immediatelyfast: ImmediatelyFast 1.2.12+1.20.5
			net_lenni0451_reflect: Reflect 1.3.2
		invmove: InvMove 0.8.6
		iris: Iris 1.7.0+mc1.20.6
			io_github_douira_glsl-transformer: glsl-transformer 2.0.0-pre13
			org_anarres_jcpp: jcpp 1.4.14
			org_antlr_antlr4-runtime: antlr4-runtime 4.11.1
		java: OpenJDK 64-Bit Server VM 21
		lambdynlights: LambDynamicLights 2.3.4+1.20.4
			pride: Pride Lib 1.2.0+1.19.4
			spruceui: SpruceUI 5.0.3+1.20.2
		lazydfu: LazyDFU 0.1.3
		lithium: Lithium 0.12.3
		minecraft: Minecraft 1.20.6
		modmenu: Mod Menu 10.0.0-beta.1
		nbttooltip: NBT Tooltip 1.8.1
		norefreshscroll: NoRefreshScroll 1.0.7
		rrls: Remove Reloading Screen 4.0.4+mc1.20.5-fabric
		servercountryflags: Server Country Flags 1.9.4
		serverpingerfixer: Server Pinger Fixer 1.0.4
		sodium: Sodium 0.5.8+mc1.20.6
		tia: Tiny Item Animations 1.20-1.1
		truesight: True Sight 1.1.0
		viafabric: ViaFabric 0.4.14+68-main
			org_yaml_snakeyaml: snakeyaml 2.2
			viafabric-mc1206: ViaFabric for 1.20.5/6 0.4.14+68-main
			viaversion: ViaVersion 4.10.0
		yet_another_config_lib_v3: YetAnotherConfigLib 3.4.2+1.20.5-fabric
			com_twelvemonkeys_common_common-image: common-image 3.10.0
			com_twelvemonkeys_common_common-io: common-io 3.10.0
			com_twelvemonkeys_common_common-lang: common-lang 3.10.0
			com_twelvemonkeys_imageio_imageio-core: imageio-core 3.10.0
			com_twelvemonkeys_imageio_imageio-metadata: imageio-metadata 3.10.0
			com_twelvemonkeys_imageio_imageio-webp: imageio-webp 3.10.0
			org_quiltmc_parsers_gson: gson 0.2.1
			org_quiltmc_parsers_json: json 0.2.1
		zoomify: Zoomify 2.13.3
			com_akuleshov7_ktoml-core-jvm: ktoml-core-jvm 0.5.1
			dev_isxander_settxi_settxi-core: settxi-core 2.10.6
			dev_isxander_settxi_settxi-kotlinx-serialization: settxi-kotlinx-serialization 2.10.6
	Loaded Shaderpack: (off)
	Launched Version: fabric-loader-0.15.10-1.20.6
	Launcher name: minecraft-launcher
	Backend library: LWJGL version 3.3.3-snapshot
	Backend API: AMD Radeon RX 6600 GL version 3.2.0 Core Profile Context 22.20.27.09.230330, ATI Technologies Inc.
	Window size: 1920x1009
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Universe: 400921fb54442d18
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Render Distance: 12/12 chunks
	Resource Packs: vanilla, fabric, appleskin (incompatible), betterconfig, boatiview (incompatible), cleardespawn (incompatible), clientcommands (incompatible), cloth-config, dynamic_fps, elytra_physics, entity_model_features, entity_texture_features, explosiveenhancement, extremesoundmuffler (incompatible), fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-convention-tags-v2, fabric-crash-report-info-v1, fabric-data-attachment-api-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-message-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, forcecloseworldloadingscreen, forgeconfigapiport, freecam, immediatelyfast, invmove, iris, lambdynlights, lazydfu, lithium, modmenu, nbttooltip, norefreshscroll, pride, rrls (incompatible), servercountryflags, serverpingerfixer, sodium, spruceui, tia, truesight, viafabric, viaversion, yet_another_config_lib_v3 (incompatible), zoomify
	Current Language: en_us
	Locale: en_US
	CPU: 8x 12th Gen Intel(R) Core(TM) i3-12100F
```

**Comment by Khajiitos** [comment]
This mod doesn't support 1.20.6 yet but I will start working on it once Forge releases for 1.20.6.

**Comment by Khajiitos** [comment]
I'm going to close this issue as I've released 1.20.6 versions for NeoForge and Fabric, unfortunately due to issues that Forge 1.20.6 created I wasn't able to make it work for it (I think this is the reason there aren't many Forge 1.20.6 mods).
